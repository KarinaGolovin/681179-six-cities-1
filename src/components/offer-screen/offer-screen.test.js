import React from 'react';
import OfferScreen from './offer-screen.jsx';
import ShallowRenderer from 'react-test-renderer/shallow';

const renderer = new ShallowRenderer();

it(`OfferScreen correctly renders`, () => {
  const component = renderer.render(
      <OfferScreen />
  );

  expect(component).toMatchSnapshot();
});
