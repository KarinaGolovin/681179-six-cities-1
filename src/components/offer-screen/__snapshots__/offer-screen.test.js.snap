// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferScreen correctly renders 1`] = `
<Memo(ConnectFunction)>
  <Memo(ConnectFunction) />
</Memo(ConnectFunction)>
`;
